unlonely-alpha/web/src/apiClient/client.ts

```diff
@@ -66,10 +63,7 @@ export function initializeApollo(
   _apolloClient.setLink(
     contextLink.concat(authLink).concat(
       new HttpLink({
-        uri:
-          process.env.NODE_ENV === "production"
-            ? server
-            : "http://localhost:4000/graphql",
+        uri: "https://sea-lion-app-j3rts.ondigitalocean.app/graphql",
```

Comment:
change this back to

`process.env.NODE_ENV === "production" ? server : "http://localhost:4000/graphql",`